x-pack/plugins/secops/public/components/timeline/timeline.tsx

```diff
@@ -110,6 +125,18 @@ export const Timeline = pure<Props>(
           </EventsQuery>
         );
       })}
+      {dataProviders.length !== 0 && (
```

Comment:
Since the `JSX` returned here will always be truthy, could this be an accidental use of `&&`?